tests/unit_tests/test_lobbyconnection.py

```diff
@@ -433,6 +471,41 @@ def test_send_mod_list(mocker, lobbyconnection, mock_games):
     assert len(bans) == 0
 
 
+async def test_command_admin_closelobby_with_ban_duration(mocker, lobbyconnection, db_engine):
```

Comment:
Re-definition of the same test function?

Reply:
(no ?)